templates/kubernetes/managed.html

```diff
@@ -0,0 +1,166 @@
+{% extends "kubernetes/base_kubernetes.html" %}
+{% block title %}Managed Kubernetes | Kubernetes{% endblock %}
+{% block meta_description %}With managed kubernetes, Canonical will build, support and manage your cluster for only $4 per node per day.{% endblock meta_description %}
+{% block extra_body_class %}kubernetes{% endblock %}
+
+
+{% block content %}
+
+  <section class="p-strip is-deep is-bordered">
```

Comment:
I think this should be `p-strip--accent is-deep` like on [cloud/managed-cloud](http://0.0.0.0:8001/cloud/managed-cloud)